### PR TITLE
Feature/track process stf

### PR DESCRIPTION
**Pull request by guiguem**
Solving issue #131 
There is a new slot for SWFC data only, the double cuts algorithm cannot be used with this slot (but a message is provided).
There is a testing script for this processor: Source/Executables/Validation/TestTrackProcessing.cc
It seems to be working.

### CODE REVIEW

File: Source/EventAnalysis/KTTrackProcessing.cc [hunk 1]
```diff
@@ -1,8 +1,9 @@
-/*
- * KTTrackProcessing.cc
- *
- *  Created on: July 22, 2013
- *      Author: N.S. Oblath & B. LaRoque
+/**
+ @file KTTrackProcessing.hh
```
Comment (cclaessens) [inline Source/EventAnalysis/KTTrackProcessing.cc]: .cc?

File: Source/Executables/Validation/TestTrackProcessing.cc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        return sftData;
+    }
+    double timeStep = (trackLength)/(nSlices-1);
+    double sliceTime = trackStart;
+    for (unsigned iSlice=0; iSlice <nSlices; ++iSlice){
+        int nPoints = numberPointsDistribution();
+        for (unsigned iPoint = 0; iPoint<nPoints; ++iPoint){
+            double yPoint = trackIntercept + trackSlope*sliceTime + noiseDistribution();
+            double power = powerDistribution();
+            Point aPoint(sliceTime,yPoint,power,sliceTime);
```
Comment (guiguem) [inline Source/Executables/Validation/TestTrackProcessing.cc]: This will fail with the #142 
Need to fix it!

**Comment by guiguem** [comment]
Change of plan: we will have two slots (SWFC and SeqLines).
For both, we will extract the Points object and use it for the weighted-slope algorithm.

**Comment by guiguem** [comment]
In the end I think Nick and I have solved #132.
Solving #131 might be worth another PR/branch.

**Pull request merged**